update_database_scripts/README_master_script.txt

```diff
@@ -1,28 +1,36 @@
+	c. 2>&1, which ensures the output of the commands will go to the file specified and not to an email
+
+----Usage Without the Cron Job----
+To update the PRSKB database manually (without using the cron job) with the latest studies from the NHGRI-EBI GWAS Catalog, follow the following steps:
 1. ssh into the PRSKB server at lsprs.byu.edu with your username and password.
 2. Navigate to the scripts folder where the website is stored: cd /var/www/prs.byu.edu/html/update_database_scripts/
 3. Open a new tmux window by typing the following command: "tmux new -s session_name" where session_name is the name of the session you are starting.
    Opening a tmux window allows the scripts to run even when you are not logged onto the server.
     a. If tmux is not available, install it with the following command: "sudo apt-get install tmux"
     b. You can leave or come back to a tmux window at any time with "tmux detach" and "tmux a -t session_name" respectively. 
-4. Run the following command, replacing "password" with the password for the PRSKB MySQL password: sudo ./master_script.sh "password" 8 &> output.txt &
+4. Run the following command, replacing "password" with the password for the PRSKB MySQL database or the path to the JavaScript file contaning the PRSKB MySQL password: 
+   sudo ./master_script.sh "password" 8 &> output.txt &
     a. After running the command, make sure the script has started working by opening the output.txt file and confirming there are no errors.
         aa. If you get the error: "./master_script.sh: Permission denied" then do "chmod 777 master_script"
     b. See the master_script file for additional information on its parameters.
     c. This command will run in the background and write its terminal output to "output.txt" Remember, while in a tmux window, run "tmux detach" and the 
        command will continue to run in the detached tmux window.
     d. The "8" in the command is the number of times the GWAS catalog will be split to download different parts of the database concurrently. 
        Increase this number to speed up the download, but don't increase the number too much since the website computer only has 2 cores.
-5. This command will take at least 4 hours to run. It is done when the last line of "output.txt" says "Press [Enter] key to finish..." At this point, 
-   all of the new data is already uploaded to the database and and you just need to upload the new TSVs to Github.
-6. Upload the new TSVs to github: (This may be another script in the future)
-    a. git add .
-    b. git commit -m "updated database *MM/DD/YY*"
-    c. git push origin master
-7. If you run the master script, but then want to stop it, you can use the following process:
+5. This command will take between 3-5 hours to run. It is done when the last line of "output.txt" says "Press [Enter] key to finish..." At this point, 
```

Comment:
Is this still between 3-5 hours?

Reply:
Yes, but it's the whole master script, not just the associations table. The associations table alone used to take 6+ hrs. Made a clarification in the documentation.